deploy/role.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
 rules:
   - apiGroups:
       - ""
+      - apps
```

Comment:
We have other rules for "apps" below.

Reply:
nice catch! the `statefulset` needs to be set there. I'm going to update it.